scss/components/_menu.scss

```diff
@@ -437,6 +458,19 @@ $menu-item-background-hover: $light-gray !default;
     }
   }
 
+  @if $menu-centered-back-compat {
+    .menu-centered {
+      > .menu {
+        @if $global-flexbox {
+          display: flex;
```

Comment:
Why do we need to add display flex? In global flexbox case won't we already be display flex?